fdbserver/VersionedBTree.actor.cpp

```diff
@@ -7941,6 +8577,7 @@ TEST_CASE("/redwood/correctness/btree") {
 
 	state int pageSize =
 	    shortTest ? 200 : (deterministicRandom()->coinflip() ? 4096 : deterministicRandom()->randomInt(200, 400));
+	state int pagesPerExtent = SERVER_KNOBS->REDWOOD_DEFAULT_EXTENT_PAGES;
```

Comment:
choose a random number  - not uniform - make 1 more popular. Choose a page size to force a single queue entry per page